spec/features/courses/confirmation_spec.rb

```diff
@@ -286,6 +286,20 @@
       include_examples "goes to the edit page"
     end
 
+    context "age range showing" do
+      let(:level) { "primary" }
+      it "does show age range if course is primary" do
+        expect(course_confirmation_page.details).to have_age_range
+      end
+    end
+
+    context "age range not showing" do
+      let(:level) { "further_education" }
+      it "does not show age range if course is further education" do
+        expect(course_confirmation_page.details).to have_no_age_range
+      end
+    end
```

Comment:
These do feel more like view specs but this is an existing problem we have with view specs being used in feature specs.

Reply:
I guess it would be some tech debt then...should I make them view specs now?

Reply:
Don't worry about it here, this PR just needs a review from @conorlyster or @jesseyuen